.github/workflows/release-phar.yml

```diff
@@ -26,36 +26,12 @@ jobs:
       - name: "Checkout"
         uses: "actions/checkout@v3"
 
-      - name: "Install PHP"
-        uses: "shivammathur/setup-php@2.24.0"
-        with:
-          coverage: "pcov"
-          php-version: "${{ matrix.php-version }}"
-          ini-values: memory_limit=-1
+      - uses: DeterminateSystems/nix-installer-action@main
```

Comment:
Is there a pinnable version for this?

Reply:
Sure, I'll do it, I'm busy fixing something inside the flake!

Reply:
OK, everything fixed, I was missing the `installPhase` ! Dumb me.

Reply:
Yes, but I'm still wondering if we can avoid referencing `@main` :D

Reply:
Yes, I understood, it's already removed :)